python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -536,6 +536,7 @@ def my_function():
                 asset1_result, asset2_result = do_some_transformation(asset0_value)
                 write(asset1_result, path="asset1")
                 write(asset2_result, path="asset2")
+                return None, None
```

Comment:
must have missed the swap from a set to a list when specifying `deps` in this (see line 532) - can you update to `deps=["asset0"]`?